grafana-plugin/src/pages/integration/IntegrationCommon.config.ts

```diff
@@ -4,21 +4,6 @@ export const TEXTAREA_ROWS_COUNT = 4;
 export const MAX_CHARACTERS_COUNT = 50;
 
 // Mostly used for input fields where we're hiding scrollbars
-export const MONACO_OPTIONS = {
```

Comment:
These lines cannot be removed as the Monaco Editors rely on 2 different configs.

Reply:
done